astropy/io/ascii/cds.py

```diff
@@ -348,26 +348,28 @@ def write_byte_by_byte(self):
 
         Example::
 
-        --------------------------------------------------------------------------------
+        ================================================================================
```

Comment:
That example dated back to the time we were the _pycdsreadme_ template for the ReadMe.
Changed to `=` for header since the example on MRT standard homepage uses it.